src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -12,47 +12,42 @@ import { getRemoteRoutePaths } from '../../common';
 import { FlameGraph } from './flamegraph';
 import { newProjectTimeQuery } from './mappings';
 
-function getSampledTraceEventsIndex(
+// Return the index that has between sampleSize..sampleSize*5 entries.
+// The starting point is the number of entries from the profiling-events-5pow6 index.
+export function getSampledTraceEventsIndex(
   sampleSize: number,
-  sampleCountFromSmallestTable: number
+  sampleCountFromPow6: number,
+  power: number
 ): [string, number, number] {
```

Comment:
Could we use a type or interface instead of an unlabeled array for the return type?

I think this would help with documentation but also would allow us to throw a type error if the two `number`s in the returned arrays are swapped.

Reply:
Amended